src/backend/crew_ai/cleaned_llm_wrapper.py

```
@@ -39,14 +49,140 @@
+        # --- model ---
+        model: str = kwargs.get("model", "unknown")
+
+        # --- prompt text — serialize messages list as JSON for queryability ---
+        prompt_text: str | None = None
+        messages = kwargs.get("messages")
+        if messages:
+            try:
+                import json as _json
+                prompt_text = _json.dumps(messages, ensure_ascii=False)
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)